README.md

```diff
@@ -1,8 +1,71 @@
+
+## Skills
+
+**Proficient:** Java, Python, JavaScript, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Linux/Bash, HTML, Docker, Machine Learning, Computer Hardware/Assembly
+
+**Familiar:** C++, Neural Networks, Networking, Cloud Computing (MS Azure/IBM Watson), Computer Vision
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_MitchellActon](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon)
+
+
+**Linux Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon/tree/master/linux_sql)]: Developed a Linux cluster hardware resource management tool intended to be used by Linux system administrators. Includes bash and SQL scripts to automatically set up the docker PostgreSQL container and database, initialize tables, and scripts to populate the tables with host hardware specification and resource usage data.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon/tree/master/core_java)]:
+      
+  - Twitter App: Under Development
+  - JDBC App: Under Development
+  - Grep App: Under Development
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon/tree/master/cloud_devops)]: Not Started
+
+
+## Highlighted Projects
+**Algorithm and Simulation Visualizer Website** [[GitHub](https://mitchellacton.github.io/WebPortfolio/site/)]: Created an interactive website to host various sorting and pathfinding algorithm visualizers, as well as some creative simulations written in JavaScript.
+
+**City Neighbourhood Clustering Algorithm**: Built an unsupervised machine learning tool in Python to cluster areas (postal/zip codes) in a given city based on the area's facilities (restaurants, green space, schools, etc.). The algorithm resulted in 5 general clusters of postal codes (suburban, urban, industrial, downtown, airport). Analyzing the data revealed the most popular facilities in these clusters (eg. Suburban areas were high in schools, green space, and yoga studios). The tool's intended use was to pinpoint the best area for a potential business owner to open their business (Suburban area with no yoga studios would have a demand for a new yoga studio).
```

Comment:
![image](https://user-images.githubusercontent.com/58951265/134979235-a2b0943e-45b3-4944-8792-6e58bafdfe68.png)
run a grammarly check please

Reply:
have you used any libraries or frameworks? did you use any IDE or notebook?